### PR TITLE
feat: /currency endpoint to query all metagraphs

### PR DESCRIPTION
**Pull request by kpudlik**

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -185,6 +185,12 @@ functions:
       - httpApi:
           path: /currency/{identifier}/addresses/{address}/fee-transactions/received
           method: GET
+  metagraphs:
+    handler: src/handler.metagraphs
+    events:
+      - httpApi:
+          path: /currency
```
Comment (scasplte2) [inline serverless.yml:192]: just my opinion but `/currency` seems somewhat confusing to retrieve a list of metagraphs that may not include a token (like a data application without fees). Two points I can think of to support this approach would be that 1) we imagine all metagraphs to include a custom token or 2) this is keeping with the historical approach for how metagraphs are labeled in other portions of the overall protocol (like the ML0 is technically a CurrencyL0). 

Not a strong preference to change this, just sharing my thoughts.

**Review by scasplte2 — APPROVED** [review]

File: serverless.yml (hunk 1 above)
Comment (marcinwadon) [reply serverless.yml:192]: @scasplte2 What do you mean by "token"?

Comment (marcinwadon) [reply serverless.yml:192]: ah, I see what you mean. I think it should be discussed wider on slack and changed across all repositories if needed. unless it's totally new endpoint, but looks like it's an aggregation of what we already return in other `/currency` endpoints

**Review by marcinwadon — APPROVED** [review]

File: serverless.yml (hunk 1 above)
Comment (kpudlik) [reply serverless.yml:192]: > just my opinion but `/currency` seems somewhat confusing to retrieve a list of metagraphs that may not include a token (like a data application without fees). Two points I can think of to support this approach would be that 1) we imagine all metagraphs to include a custom token or 2) this is keeping with the historical approach for how metagraphs are labeled in other portions of the overall protocol (like the ML0 is technically a CurrencyL0). 
> 
> 
> 
> Not a strong preference to change this, just sharing my thoughts.

Fair point. The /currency was just a part of the ticket but I think we should make /metagraphs or something

**Review by marcus-girolneto — APPROVED** [review]

**Pull request merged**